src/pages/Map/Map.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { compose, withProps } from 'recompose';
+import { withScriptjs, withGoogleMap, GoogleMap, Marker } from 'react-google-maps';
+
+import Header from '../../components/Header/Header';
+
+const MyMapComponent = compose(
+  withProps({
+    googleMapURL: 'https://maps.googleapis.com/maps/api/js?v=3.exp&libraries=geometry,drawing,places',
+    loadingElement: <div style={{ height: `100%` }} />,
+    containerElement: <div style={{ height: `600px` }} />,
+    mapElement: <div style={{ height: `100%` }} />,
+  }),
+  withScriptjs,
+  withGoogleMap)((props) =>
+  <GoogleMap
+    defaultZoom={16}
+    defaultCenter={{ lat: 53.120328, lng: 8.679901 }}
+  >
+    {props.isMarkerShown && <Marker position={{ lat: 53.120328, lng: 8.679901 }} onClick={props.onMarkerClick} />}
```

Comment:
there might be a list of markers.

Reply:
True, I just put that already in there just to get an idea of how the markers work.